usr/src/uts/common/fs/zfs/metaslab.c

```diff
@@ -199,6 +200,43 @@ static void metaslab_set_fragmentation(metaslab_t *);
 kmem_cache_t *metaslab_alloc_trace_cache;
 
 /*
+ * How many TXG's worth of updates should be aggregated per TRIM/UNMAP
+ * issued to the underlying vdev. We keep two range trees of extents
+ * (called "trim sets") to be trimmed per metaslab, the `current' and
```

Comment:
Btw, I'm apparently not the only one who uses that style. It seems you've used it as well. See zfs_vnops.c:2307:
```* Special case `.', `..', and `.zfs'. ```
Anyway, if necessary, I'll be happy to replace these with simple quotes.